### PR TITLE
LPS-161710 | master

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LPS-161710

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 1]
```diff
@@ -212,6 +212,64 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-150460 TC-1. User is able to trigger propagation of the site template to all connected sites automaticly."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:215]: "automaticly" I'm not sure if this is an existing word.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 2]
```diff
@@ -397,6 +455,62 @@ definition {
...
+				siteName = "Site Template Site Name",
+				siteTemplateName = "Site Template Name",
+				siteType = "Site Template");
+
+			SiteTemplates.addSitePageViaSiteTemplateCP(
+				pageName = "Test",
+				siteTemplateName = "Site Template Name");
+		}
+
+		task ("When: User visits the Site Template and verifies that the page is not present on the Site based on the database") {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:484]: "via database look up"

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 3]
```diff
@@ -397,6 +455,62 @@ definition {
...
+				siteTemplateName = "Site Template Name");
+
+			var sqlResults = SQL.executeMySQLStatement(mysqlStatement = "${sQLStatement}");
+
+			TestUtils.assertNotEquals(
+				actual = "${sqlResults}",
+				expected = "${consoleText}");
+		}
+
+		task ("And: Open the site based on the template, triggering the propagation") {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:497]: "to trigger propagation"

**Review by gaborlovasdotcom — COMMENTED** [review]
Please see my comments, the test flow is ok.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase (hunk 1 above)
Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:215]: Changed to: automatically

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase (hunk 2 above)
Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:484]: Changed to: via looking up the database

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**